recipes/metamorpheus/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.0.316" %}
+{% set sha256 = "389c7446abe61a931a78f3c27478baadf6cb43d9a00840b954c22a3ec429a316" %}
+
+package:
+  name: metamorpheus
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/smith-chem-wisc/MetaMorpheus/releases/download/{{ version }}/MetaMorpheus_CommandLine.zip
+  sha256: '{{ sha256 }}'
+
+build:
+  skip: true  # [osx or linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  run:
+    - dotnet =3.1
```

Comment:
It doesn't work with 5?

Reply:
Not yet, but that's in the works. https://github.com/smith-chem-wisc/MetaMorpheus/pull/2047

Reply:
I'll try removing it and see what happens and revert if needed. I think I remember that if the NetCore.App 3.1 runtime isn't present, it doesn't succeed.

Reply:
This is what I see with 5.0:

![image](https://user-images.githubusercontent.com/16342951/113702665-1daf5980-96da-11eb-9b5a-0f29569f80f3.png)

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=300108&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77